packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -370,49 +398,49 @@ export class DevWorkspaceClient extends WorkspaceClient {
       channel: 'onModified'
     };
     await this.websocketClient.subscribe(message);
-    this.websocketClient.addListener(message.channel, (devworkspace: IDevWorkspace) => {
+    this.websocketClient.addListener(message.channel, (maybeDevworkspace: unknown) => {
+      if (isWorkspaceV2(maybeDevworkspace as any) === false ) {
+        console.warn(`Channel "${message.channel}" received object that is not a devWorkspace, skipping it: `, maybeDevworkspace);
```

Comment:
warning is not visible enough. We must propagate it to UI to detect issues earlier.

Reply:
Ok. I have improved it.